src/components/ProductMilestoneCloseResultDetailPage/ProductMilestoneCloseResultDetailPage.tsx

```diff
@@ -41,6 +44,8 @@ export const ProductMilestoneCloseResultDetailPage = () => {
 
   const closeResult: ProductMilestoneCloseResult | undefined = serviceContainerProductMilestoneCloseResult.data?.content?.[0];
 
+  const [logBuffer, addLogLines] = useDataBuffer(100, timestampHiglighter);
```

Comment:
also please increase the delay to 500ms (or if you find a higher value more appropriate, tell me)

Reply:
set to 750 as Martin mentioned here: https://github.com/project-ncl/pnc-web-ui-react/pull/577#discussion_r1778191641